now_playing.py

```diff
@@ -58,4 +58,4 @@ def foreach_window(hwnd, lParam):
 # Enumerate all of the windows looking for Spotify.
 EnumWindows(EnumWindowsProc(foreach_window), 0)
 
-print titles[0] if len(titles) == 1 else 'No song playing...'
+print (titles[0] if len(titles) == 1 else 'No song playing...')
```

Comment:
Looks like this is mainly a fix to get it working on python 3.

nit-pick: Remove the space after `Print`. e.g. `Print(....)`

Reply:
Just trying to help 🙃